README.md

```diff
@@ -160,7 +160,7 @@ kernel). Users are also able to replace it with a custom verifier.
 For `struct EbpfVmFixedMbuff`, two additional arguments must be passed to the
 constructor: `data_offset` and `data_end_offset`. They are the offset (byte
 number) at which the pointers to the beginning and to the end, respectively, of
-the memory area of packet data are to be stored in the internal metadata buffer
+the memory area of the packet data, are to be stored in the internal metadata buffer
```

Comment:
Nack for this one. it's “the offset at which the pointers (to the start and end (of memory area (of packet data))) are to be stored”. Adding a comma does not sound right?